src/assets/v1.3.0/sign_message_lib.json

```diff
@@ -242,6 +242,7 @@
     "713715": ["eip155", "canonical"],
     "764984": "canonical",
     "810180": "zksync",
+    "978657": "0xEb4b222B3B89Ce6dbE14012432a223D2Faa31D06",
```

Comment:
👆

Reply:
@remedcu when we deployed with the branch 1.3.0 of https://github.com/safe-global/safe-smart-account/tree/v1.3.0? when i deployed all smart contract corresponding with the Safe Contract Address Canonical all!!

Reply:
@remedcu can you support me about this issue!, because i use the script "yarn deploy-all custom" in the repo: https://github.com/safe-global/safe-smart-account/tree/v1.3.0, version 1.3.0, and deployed all Safe Contract with the "canonical" address except this one?

Reply:
Fixed

Reply:
@remedcu pls Check again sorry!!